ocrd_validator.py

```diff
@@ -124,7 +124,7 @@ def __validate_single_len_token_symbols(self, line_index, token_index, token):
         if token in (QM, BACKSLASH):
             return True
         else:
-            info = f"TOKEN_ERROR01: Invalid token: {token}"
+            info = f"TOKEN_ERROR_RULE_01: Invalid token: {token}"
```

Comment:
I think we need a distinction between symbol validation rules and just validation rules. I would change "TOKEN_ERROR_RULE_01" to "TOKEN_SYMBOL_ERROR_RULE_01"